Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
```

Comment:
For local testing I use https://github.com/jenkinsci/filesystem_scm-plugin/pull/2.
For upstream PR testing I would like to get it hosted first :)